rcongui/README.md

```diff
@@ -1,68 +1,82 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# CRCON GUI
 
-## Available Scripts
+A React-based web interface for managing Hell Let Loose game servers.
 
-In the project directory, you can run:
+## Tech Stack
 
-### `yarn start`
+- React 18
+- Vite
+- React Router 6
+- Material UI 6
+- Tanstack Query 5
+- Tanstack Table 8
+- Zustand
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Prerequisites
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
+- Node.js (v18 or higher)
+- NPM package manager
+- A running CRCON server instance
 
-### `yarn test`
+## Setup
 
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+1. Clone the repository
+2. Navigate to the project directory
 
-### `yarn build`
+```bash
+cd rcongui
+```
 
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
+3. Install dependencies:
 
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
+```bash
+npm install
```

Comment:
shouldn't this better be `npm ci` as long as one don't want to add/change dependencies? 🤔

Reply:
Uhm, yeah, and? 😅 Sorry, I really don't get it :D The `package-lock.json` still includes all the development packages. When running `npm ci` without the `--omit=dev` flag, then all packages will be installed :)

Just want to make sure that people do not get changes in their working tree unintentional.